### PR TITLE
COMMERCE-8754 Placed Orders Widget display date configuration

### PR DESCRIPTION
**Pull request by ghost**
https://issues.liferay.com/browse/COMMERCE-8754

### CODE REVIEW

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:stop

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by michaelhashimoto** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/util/CommerceOrderClayTableUtil.java [hunk 1]
```diff
@@ -61,6 +63,9 @@
  */
 public class CommerceOrderClayTableUtil {
 
+	public static final String DATE_AND_TIME_FORMAT =
+		"MMM dd, yyyy hh:mm:ss aa";
+
```
Comment (ivicac) [inline modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/util/CommerceOrderClayTableUtil.java:68]: @CrescenzoRegaLF do we need this constant public?

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/portlet/configuration/CommerceOrderContentPortletInstanceConfiguration.java [hunk 2]
```diff
@@ -38,4 +38,7 @@
 	@Meta.AD(deflt = "0", name = "display-style-group-id", required = false)
 	public long displayStyleGroupId();
 
+	@Meta.AD(deflt = "true", name = "show-time", required = false)
+	public boolean showTime();
+
 }
```
Comment (ivicac) [inline modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/portlet/configuration/CommerceOrderContentPortletInstanceConfiguration.java:44]: @CrescenzoRegaLF can we please call it showCommerceOrderCreationTime?

File: modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/util/CommerceOrderClayTableUtil.java (hunk 1 above)
Comment (ivicac) [inline modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/util/CommerceOrderClayTableUtil.java:68]: @CrescenzoRegaLF can we please replace this with

_commerceOrderDateFormatDate = FastDateFormatFactoryUtil.getDate(
   DateFormat.MEDIUM, themeDisplay.getLocale(),
   themeDisplay.getTimeZone());
_commerceOrderDateFormatTime = FastDateFormatFactoryUtil.getTime(
   DateFormat.MEDIUM, themeDisplay.getLocale(),
   themeDisplay.getTimeZone());

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci:forward

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by ivicac** [comment]
ci>forward

**Pull request closed without merging**